### PR TITLE
Feat(axios extension): Load service annotations without service catalog request

### PR DESCRIPTION
**Pull request by IainSAP**
https://github.com/SAP/open-ux-tools/issues/1992

- Allows annotation retrieval for a specific service without loading all services from the catalog (which adds an extra network call and cant be used with mock server)
- Adds patching of certs for sap systems

### CODE REVIEW

File: packages/axios-extension/src/abap/catalog/v2-catalog-service.ts [hunk 1]
```diff
@@ -138,8 +145,11 @@ export class V2CatalogService extends CatalogService {
      * @param filter.path filter by path
      * @returns service annotations
      */
-    protected async getServiceAnnotations({ id, title, path }: FilterOptions): Promise<ODataServiceV2Info[]> {
-        if (!this.entitySet) {
+    protected async getServiceAnnotations(
+        { id, title, path }: FilterOptions,
+        preloadCatalog = true
```
Comment (tobiasqueck) [inline packages/axios-extension/src/abap/catalog/v2-catalog-service.ts:150]: Getting the correct entity set is not about preloading, it is about fining the correct services depending on the version of the ABAP backend.

Comment (tobiasqueck) [reply packages/axios-extension/src/abap/catalog/v2-catalog-service.ts:150]: Looking at it again, I see your point. It is required for fetching the services, however, for fetching the annotation of a specific service you can always use `ServiceCollection`, so there is no need to add this preload parameter but you can remove the  `this.determineEntitySet` call from and always use `/ServiceCollection` as the base path.

Comment (IainSAP) [reply packages/axios-extension/src/abap/catalog/v2-catalog-service.ts]: Done thanks

**Review by tobiasqueck — CHANGES_REQUESTED** [review]
* changeset missing
* could you please add a comment to the global-root cert file where you got it from (just one a clear reference to the publicly available file so that nobody gets confused and assumes there is something secret

**Review by devinea — APPROVED** [review]
Code changes and tests look good.
Changeset ✅

**Review by cianmSAP — APPROVED** [review]
Changes are clear 
Coverage above 80
Changeset ✅

**Pull request merged**